cmd/clusterctl/hack/create-local-repository.py

```diff
@@ -68,19 +70,19 @@
         'configFolder': 'controlplane/kubeadm/config/default',
     },
     'infrastructure-docker': {
-        'componentsFile': 'infrastructure-components.yaml',
+        'componentsFile': 'infrastructure-components-development.yaml',
         'nextVersion': 'v1.7.99',
         'type': 'InfrastructureProvider',
         'configFolder': 'test/infrastructure/docker/config/default',
     },
     'infrastructure-in-memory': {
-          'componentsFile': 'infrastructure-components.yaml',
+          'componentsFile': 'infrastructure-components-development.yaml',
```

Comment:
If I see correctly this should be "artifacts.k8s-staging-cluster-api.appspot.com/components/nightly_main_20240101/infrastructure-components-in-memory-development.yaml"

Reply:
Nice catch, fixed